### DISCUSSION TITLE
Importation squirrel sql connexions

### DISCUSSION THREAD

**Discussion by ccavallie** [dbeaver/dbeaver#22824] [Q&A]
Hello,
I would like to import all SQL squirrel connections (v3.7.1) into dbeaver (v 22.1.3.202207240747). I exported the file in SQL squirrel xml format. I get the message "Connection list is empty" without being able to select the file I generated from SQL squirrel and I don't know where to copy this file so that the import with "third party Configuration" and the SQL Squirrel wizard can find this file.
Can you help me solve this problem?

**Comment by LonwoLonwo** [comment]
Hello @ccavallie 

So, does this import not work for you?

![2024-03-19 11_13_44-Window](https://github.com/dbeaver/dbeaver/assets/45152336/2c451874-e9be-49f0-b84d-2bd708019971)

**Comment by nfurquez** [comment]
Hi, I have dBeaver v24.0.2, and have the same problem. I can't import anything
The Squirrel files must be in specific path? I don't have installed Squirrel.
![image](https://github.com/dbeaver/dbeaver/assets/2037318/e51b8a7a-8e50-4bef-82c3-b4ea86cd2fce)

**Reply by rawyler** [reply]
The path which is checked is `$HOME/.squirrel-sql/` whereas the two required files are `SQLAliases23.xml` and `SQLDrivers.xml`. Both files have to be present to make it work.
See: https://github.com/dbeaver/dbeaver/blob/7da536042bb037a1b4a4983477e4d140478a384b/plugins/org.jkiss.dbeaver.ext.config.migration/src/org/jkiss/dbeaver/ext/config/migration/wizards/squirrel/ConfigImportWizardPageSquirrel.java#L41
